src/Logic/Utilities.cs

```diff
@@ -1559,7 +1566,7 @@ public static string AutoDetectLanguageName(string languageName, Subtitle subtit
                         }
                         break;
                     case "hr_HR": // Croatian
-                        count = GetCount(text, AutoDetectWordsCroatian);
+                        count = GetCount(text, AutoDetectWordsCroatianAndSerbian);
                         if (count > bestCount)

```

Comment:
It will but it's only temporary until Serbian detection is added as I said in my first comment. I don't have the knowledge to implement it myself. Besides, the current list also contains words from both languages so the behaviour is still the same.